src/game/client/in_camera.cpp

```diff
@@ -45,6 +46,8 @@ enum ECAM_Command
 
 //-------------------------------------------------- Global Variables
 
+ConVar dem_forcehltv_local_model("dem_forcehltv_local_model", "0", FCVAR_BHL_ARCHIVE, "Drawing player model that recorded in 3rd person for POV demos");
```

Comment:
If I understand correctly, this cvar, when enabled, will draw player's actual model in third person mode when playing POV demos. If so, could you please change the comment to "Draw correct player model in 3rd person POV demos"?